http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -52,11 +54,12 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
   private[monitor] def messageHandler: Pipe[F, Event[LeoPubsubMessage], Unit] = in => {
     in.evalMap { event =>
       val traceId = event.traceId.getOrElse(TraceId("None"))
-
       val now = Instant.ofEpochMilli(com.google.protobuf.util.Timestamps.toMillis(event.publishedTime))
       implicit val appContext = ApplicativeAsk.const[F, AppContext](AppContext(traceId, now))
       val res = for {
-        res <- messageResponder(event.msg, now).attempt
+        res <- messageResponder(event.msg, now)
+          .timeout(config.timeout)
```

Comment:
Trying to understand `timeout` semantics. As I understand it if a timeout occurs it will throw `TimeoutException` and will end up matching this case:
```
case _ =>
  logger.error(e)("Fail to process non-retryable pubsub message") >> ack(event)
```

Maybe we should add another case to specifically handle timeouts.

Also if a timeout occurs is the underlying `IO` cancelled or does it keep running? Thinking 55 seconds might be too short to fail cluster creations if some Google calls retry.

Reply:
we've been using 1 min as subscriber's ack timeout. What do you think should happen if time out happens other than logging the error and acking the message?

We can bump both the subscriber's ack deadline and this timeout if 1 min is not enough.

According to cats-effect doc, the underlying IO is canceled in the event of time out

Reply:
Yeah agreed, not much we can do besides log the error. Maybe trigger a stackdriver alert?

I think 1 minute is probably too short -- a single Google call can retry for more than 1 minute.  Maybe 5 minutes?

Reply:
yea, I wish we've set up the ability to error reporting (like if we have sentry set up properly, anything logged as `error` will go to alert). Added https://broadworkbench.atlassian.net/browse/IA-1857.

I'll up the ack deadline